package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@primer/blueprints",
-  "version": "4.0.0",
```

Comment:
Why does it say the version here is 4.0.0, but when I go into the Code tab it says the version is 4.0.1 in this repo 🤔

Reply:
https://github.com/primer/blueprints/blob/master/package.json